bin/linguist

```diff
@@ -77,8 +105,10 @@ else
   Linguist v#{Linguist::VERSION}
   Detect language type for a file, or, given a directory, determine language breakdown.
 
-  Usage: linguist <path>
-         linguist <path> [--breakdown] [--json]
-         linguist [--breakdown] [--json]
+  Usage: linguist [path] [--breakdown] [--json] [-b]
+  [path] is the path to the git repository.
+  [--breakdown] lists the files in tabular form.
+  [--json] dumps data in JSON format
+  [-b] displays file size with unit, depending on the file size.
```

Comment:
The `df` and `ls` utilities also use `-h` for human-readable output. I'd strongly recommend the same.